screens/Authentication/SignUp/CreateCredentials.tsx

```diff
@@ -68,13 +69,29 @@ export default function CreateCredentials() {
     },
   }
 
+  const updateUsername = async () => {
+    await api.patch("/auth/update-profile", {
+      username: text,
+    })
+    // TODO: navigate to the noFriends screeen
+    navigation.navigate("Home")
```

Comment:
Let's just navigate to the home screen and remove the `TODO`.With the currently implemented logic, we do not have a noFriends screen, we have a home page that will display different things depending on whether the user has friends or not.